tests/time_tracker_api/users/users_namespace_test.py

```diff
@@ -23,3 +29,37 @@ def test_users_response_contains_expected_props(
     assert 'name' in json.loads(response.data)[0]
     assert 'email' in json.loads(response.data)[0]
     assert 'role' in json.loads(response.data)[0]
+
+
+def test_update_user_role_response_contains_expected_props(
+    client: FlaskClient,
+    valid_header: dict,
+):
+    AzureConnection.update_user_role = Mock(
+        return_value=[{'name': 'dummy', 'email': 'dummy', 'role': 'dummy'}]
```

Comment:
I think, return  should be a single object here. e.g. : {'name': 'dummy', 'email': 'dummy', 'role': 'dummy'}